packages/flutter_tools/templates/plugin/ios-swift.tmpl/projectName.podspec.tmpl

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
should we just move ios.tmpl/projectName.podspec.tmpl to ios-objc.tmpl/projectName.podspec.tmpl then if ios-swift.tmpl is getting its own?

Reply:
It was renamed in this commit as well.
.../plugin/ios.tmpl/projectName.podspec.tmpl → ...in/ios-objc.tmpl/projectName.podspec.tmpl